Add a sentence to installation page to link the "activate license" page so enterprise users understand where to go

09jvilla opened the pull request:
Right now our installation docs mix instructions on how to install Grafana and how to install Grafana Enterprise. This makes sense since its basically the same process but just with a different image/binary. 

However none of the docs link to our "activate you're grafana enterprise license" page, which is a necessary step after you install the enterprise binary if you want to actually use Grafana Enterprise. 

Adding a one sentence change to call this out on the top level installation page.

09jvilla committed:
Updated index of installation page.

09jvilla committed:
Updated index of installation page.

09jvilla committed:
Adding a reference to link Grafana Enterprise to activate license page.

09jvilla committed:
Adding a reference to link Grafana Enterprise to activate license page.

09jvilla commented on docs/sources/installation/_index.md:
```diff
@@ -16,6 +16,9 @@ This section discusses the hardware and software requirements as well as the pro
 Refer to [Upgrade Grafana]({{< relref "upgrading.md" >}}) topic. If you need to restart Grafana, refer to the [Restart Grafana]({{< relref "Restart-grafana.md" >}})
 topic for detailed information.
+
+If you want to install Grafana Enterprise, follow the instructions for installing Grafana on your particular operating system and then refer to [Activate an Enterprise license]({{< relref "activate-license.md" >}}). 
```
The relref I'm using here is broken :/ 
I'm trying to figure out how to point to https://grafana.com/docs/grafana/latest/enterprise/activate-license/
which is under filepath docs/sources/enterprise/activate-license.md
but no luck. 

Can you help me understand what filepath our system is looking at when it does the build?

osg-grafana replied:
Is this unrelated to the merge conflict?
`
Auto-merging docs/sources/installation/_index.md
--
14 | CONFLICT (content): Merge conflict in docs/sources/installation/_index.md`

wbrowne committed:
Chore: Bump grafana-aws-sdk dep to 0.4.0 (#32563)

dsotirakis committed:
Add ability to override config variables with env variables (#32554)

* Add ability to override config variables with env variables

* Inline checkForOverrides

* Update pkg/plugins/backendplugin/manager/plugin_settings.go

Co-authored-by: Arve Knudsen <arve.knudsen@gmail.com>

Co-authored-by: Arve Knudsen <arve.knudsen@gmail.com>

torkelo committed:
Babel: Refactoring config so that it's shared and removes legacy plugin  (#32559)

* Updated plugins

* Fixing stuff

* Update

* Fixing more stuff

09jvilla committed:
Fixed relref path and minor update to copy.

09jvilla committed:
Fixed relref path and minor update to copy.

achatterjee-grafana reviewed (approved):
LGTM

papagian committed:
[Alerting]: Alertmanager API implementation (#32174)

* Add validation for grafana recipient

* Alertmanager API implementation (WIP)

* Fix encoding/decoding receiver settings from/to YAML

* Save templates together with the configuration

* update POST to apply latest config

* Alertmanager service enabled by the ngalert toggle

* Silence API integration with Alertmanager

* Apply suggestions from code review

Co-authored-by: gotjosh <josue@grafana.com>
Co-authored-by: Ganesh Vernekar <15064823+codesome@users.noreply.github.com>

ryantxu committed:
Chore: update caniuse (#32569)

oddlittlebird committed:
UI text edits (#32524)

* Update TeamSettings.tsx

* Update navModel.ts

* Update ChangePasswordForm.tsx

* Update UserProfileEditForm.tsx

* Update DashboardImportPage.tsx

* Update uploadDashboardDirective.ts

* Update ImportDashboardForm.tsx

* Update ImportDashboardOverview.tsx

* Update validation.ts

* Update PlaylistEditPage.tsx

* ui text edits

* text edits

* Update buildCategories.ts

* text edits

* text edits

* Fix formatting

* Update test snapshots

Co-authored-by: dsotirakis <sotirakis.dim@gmail.com>

torkelo committed:
TimeSrv: Refactor service to have no dependency on angular (#32562)

* TimeSrv: Refactor service to have no dependency on angular

* Fixing reference to function that does not exist

* fixing tests

* Worked around the strange error

dsotirakis committed:
Fix broken gtime tests (#32582)

torkelo committed:
Devenv: Fix prometheus config so that it scrapes your dev instance like before (#32583)

* Devenv: Fix prometheus config so that it scrapes your dev instance like before

* prom1

papagian committed:
[Alerting]: Grafana managed ruler API implementation (#32537)

* [Alerting]: Grafana managed ruler API impl

* Apply suggestions from code review

* fix lint

* Add validation for ruleGroup name length

* Fix MySQL migration

Co-authored-by: kyle <kyle@grafana.com>

torkelo committed:
Babel: Target more modern browsers, updates browserlistrc (#32573)

* Babel: Target more modern browsers, updates browserlistrc

* Updates

* Fixed circular dependencies, grafana booted

zoltanbedi committed:
Chore: Upgrade tslib (#32449)

Co-authored-by: Ryan McKinley <ryantxu@gmail.com>

jackw committed:
chore: bump execa to v2.1.0 (#32543)

* chore(toolkit): bump execa to v2 to fix vulnerability issue

* chore(toolkit): update usage of execa

osg-grafana commented on docs/sources/installation/_index.md:
```diff
@@ -16,6 +16,9 @@ This section discusses the hardware and software requirements as well as the pro
 Refer to [Upgrade Grafana]({{< relref "upgrading.md" >}}) topic. If you need to restart Grafana, refer to the [Restart Grafana]({{< relref "Restart-grafana.md" >}})
 topic for detailed information.
+
+If you want to install Grafana Enterprise, follow the instructions for installing Grafana on your system of choice and then refer to [Activate an Enterprise license]({{< relref "../enterprise/activate-license.md" >}}). 
```
```suggestion
To install Grafana Enterprise, refer to the instructions for installing Grafana on your system. Then, refer to [Activate an Enterprise license]({{< relref "../enterprise/activate-license.md" >}}).
```

torkelo committed:
Emotion: Upgrades emotion from 10 to 11 and updates all import paths  (#32541)

* Babel: Updates babel dependencies to latest version

* Emotion: Upgrade form 10 to 11

* Fixing tests

* Updated to use emotion/css instead in test

alexanderzobnin committed:
Fix access control store init (#32594)

jackw committed:
Configuration: Prevent browser hanging / crashing with large number of org users (#32546)

* fix(userslist): introduce pagination to prevent browser crash with large number of users

* test(userlist): fix failing tests

* refactor(userlist): use layout components for spacing

* test(userslist): update snapshots

leeoniya committed:
Icons: use plain SVG files, load on-demand (#32387)

torkelo committed:
Influx/Azure: Fix issue in query editors due to babel upgrade (#32605)

oddlittlebird committed:
Edit dashboard UI text (#32520)

* Update DashboardPermissions.tsx

* Update DashboardRow.tsx

* Update AutoRefreshIntervals.tsx

* Update JsonEditorSettings.tsx

* ui updates

* Update FolderPickerCtrl.ts

* Update LinkSettingsEdit.tsx

* Update LinkSettingsHeader.tsx

* Update LinkSettingsList.tsx

* Update actions.ts

* Update AngularPanelOptions.tsx

* Update getFieldOverrideElements.tsx

* Update OptionsPaneOptions.tsx

* Update types.ts

* Update usePanelLatestData.ts

* Update VisualizationButton.tsx

* Update RowOptionsModal.tsx

* Update SaveDashboardAsForm.tsx

* Update SaveDashboardForm.tsx

* Update SaveProvisionedDashboardForm.tsx

* Update SaveDashboardErrorProxy.tsx

* text edits

* Update ShareSnapshot.tsx

* Update PanelTypeCard.tsx

* Update TransformationsEditor.tsx

* Update panel.test.ts

* text edits

* Update DashboardSrv.ts

* Update VersionHistoryButtons.tsx

* Update DiffViewer.tsx

* Update VersionHistoryTable.tsx

* Update DashboardGrid.tsx

* Update DashboardPanel.tsx

* Update PanelPluginError.tsx

* Update DashboardMigrator.ts

oddlittlebird committed:
Variables UI text edits (#32523)

* Update formatRegistry.ts

* text edits

* Update adapter.ts

* Update adapter.ts

* text edits

* Update SelectionOptionsEditor.tsx

* Update CustomVariableEditor.tsx

* Update DataSourceVariableEditor.tsx

* Update IntervalVariableEditor.tsx

* Update VariableEditorList.tsx

* Update adapter.ts

* Update actions.ts

* Update NetworkGraphModal.tsx

* Update actions.ts

* Update operators.ts

* text edits

papagian committed:
[Alerting]: Delete obsolete database table and code (#32595)

* Delete obsolete migration

* Remove redundant code

ryantxu committed:
Arrow: move arrow support from frontend to backend only (#32575)

torkelo committed:
FrontendMetrics: Adds new backend api that frontend can use to push frontend measurements and counters to prometheus (#32593)

* FrontendMetrics: Adds new backend api that frontend can use to push frontend measurements and counters to prometheus

* FrontendMetrics: Adds new backend api that frontend can use to push frontend measurements and counters to prometheus

* Fix naming

* change to histogram

* Fixed go lint

kylebrandt committed:
SSE: Support time series frames with non-nullable float64 values (#32608)

* SSE: fix reduce to handle non-null
* add type for data.Field that is float64 or *float64
* resample fix for non-null input case, add couple non-null tests

achatterjee-grafana committed:
Added note to clarify version 10 and above. (#32612)

ryantxu committed:
Chore: update centrifuge import (avoid protobuf) (#32616)

torkelo committed:
GettingStarted: Fixes backgrounds in welcome and getting started (#32613)

ryantxu committed:
Live: remove measurement controller (#32622)

torkelo committed:
MySQL/Postgres/CloudMonitoring: Fixes issues related to babel upgrade (#32619)

thisisobate committed:
DataSourceList: expands the clickable area of card (#32601)

torkelo committed:
ReactSelect: Upgrades react-select from v3 to v4 and switches from fork to main npm package (#32618)

* ReactSelect: Upgrades react-select from v3 to v4 and switches from fork to main npm package

* Updated snapshot

* Removed hidden time in timepicker

Commit:
Elasticsearch: Disable Alias field for non-timeseries queries (#32279)

* Elasticsearch: Disable Alias field for non-timeseries queries

* Update public/app/plugins/datasource/elasticsearch/components/QueryEditor/index.test.tsx

dprokop committed:
TablePanel: Add solid  background cell display mode (#32446)

* Add solid color background cell display mode

* Update docs/sources/panels/visualizations/table/table-field-options.md

Co-authored-by: achatterjee-grafana <70489351+achatterjee-grafana@users.noreply.github.com>

* Update docs/sources/panels/visualizations/table/table-field-options.md

Co-authored-by: achatterjee-grafana <70489351+achatterjee-grafana@users.noreply.github.com>

kylebrandt committed:
SSE/Alerting: Fix classic condition logic and some more tests (#32615)

kylebrandt committed:
ngalert: json dataframe on temp endpoints (#32641)

kylebrandt committed:
SSE/docs: Add note on not working with dash vars (#32646)

server side expressions "Expressions" unfortunately do not work with dashboard variables (in the same way alerting does not). 

The execution path is always through the backend/server of data sources, which currently don't support expanding dashboard variables.

davidmparrott committed:
Alerting: Persist alerts on evaluation and shutdown. Warm cache from DB on startup (#32576)

* Initial commit for state tracking

* basic state transition logic and tests

* constructor. test and interface fixup

* use new sig for sch.definitionRoutine()

* test fixup

* make the linter happy

* more minor linting cleanup

* Alerting: Send alerts from state tracker to notifier

* Add evaluation time and test

Add evaluation time and test

* Add cleanup routine and logging

* Pull in compact.go and reconcile differences

* Save alert transitions and save all state on shutdown

* pr feedback

* WIP

* WIP

* Persist alerts on evaluation and shutdown. Warm cache on startup

* Filter non-firing alerts before sending to notifier

Co-authored-by: Josue Abreu <josue@grafana.com>

papagian committed:
Remove more dead code (#32645)

kylebrandt committed:
Alerting: Fix persistance migration (#32650)

FZambia committed:
Live: update Streaming plugin definitions, put frame schema in subscribe result data (#32561)

Co-authored-by: Torkel Ödegaard <torkel@grafana.org>
Co-authored-by: Ryan McKinley <ryantxu@gmail.com>

tonychoe committed:
Configure the Jaeger data source with provisioning (#32620)

* Configure the Jaeger data source with provisioning

We should have a good instruction and example for configuring the Jaeger data source with provisioning. It was hard to find how to add the tarce-to-logs, so I made one.

* Update docs/sources/datasources/jaeger.md

Co-authored-by: achatterjee-grafana <70489351+achatterjee-grafana@users.noreply.github.com>

Co-authored-by: achatterjee-grafana <70489351+achatterjee-grafana@users.noreply.github.com>

kylebrandt committed:
ngalerting: represent nil/empty labels the same (#32652)

was getting duplicates of [] and null before

ryantxu committed:
Reducers: minor reducer cleanup (#32656)

ryantxu committed:
Live: throttle messages when FPS decreases (#32627)

* throttle when FPS is low

* fix throttling

* grafanaStreamingPerfBudget

* grafanaStreamingPerfBudget

* change global strategy

* also throttle frontend

Co-authored-by: Leon Sorokin <leeoniya@gmail.com>

papagian committed:
[Alerting]: Update scheduler to evaluate rules created by the unified API (#32589)

* Update scheduler

* Fix tests

* Fixes after code review feedback

* lint - add uncommitted modifications

Co-authored-by: kyle <kyle@grafana.com>

ryantxu committed:
Panel Editor: avoid updates when schema is the same (#32668)

dependabot[bot] committed:
Bump github.com/aws/aws-sdk-go from 1.38.7 to 1.38.12 (#32682)

Bumps [github.com/aws/aws-sdk-go](https://github.com/aws/aws-sdk-go) from 1.38.7 to 1.38.12.
- [Release notes](https://github.com/aws/aws-sdk-go/releases)
- [Changelog](https://github.com/aws/aws-sdk-go/blob/main/CHANGELOG.md)
- [Commits](https://github.com/aws/aws-sdk-go/compare/v1.38.7...v1.38.12)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump github.com/go-openapi/strfmt from 0.20.0 to 0.20.1 (#32684)

Bumps [github.com/go-openapi/strfmt](https://github.com/go-openapi/strfmt) from 0.20.0 to 0.20.1.
- [Release notes](https://github.com/go-openapi/strfmt/releases)
- [Commits](https://github.com/go-openapi/strfmt/compare/v0.20.0...v0.20.1)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump github.com/hashicorp/go-version from 1.2.1 to 1.3.0 (#32687)

Bumps [github.com/hashicorp/go-version](https://github.com/hashicorp/go-version) from 1.2.1 to 1.3.0.
- [Release notes](https://github.com/hashicorp/go-version/releases)
- [Changelog](https://github.com/hashicorp/go-version/blob/master/CHANGELOG.md)
- [Commits](https://github.com/hashicorp/go-version/compare/v1.2.1...v1.3.0)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Commit:
CloudWarch: Fix service quotas link (#32686)

papagian committed:
Alerting: 404 error status when no alertmanager configuration (#32651)

oddlittlebird committed:
Update documentation-style-guide.md (#32690)

FZambia committed:
Live: expose HTTP push endpoint that will read influx line protocol and publish to websocket (#32311)

Co-authored-by: Ryan McKinley <ryantxu@gmail.com>

oddlittlebird committed:
Docs: Add Grafana basics section (#32693)

* moved files

* single sourcing

* Update _index.md

* Update heatmap.md

* Update _index.md

* Update _index.md

* Fixed 7 broken relrefs

* Fixed one more broken relref.

Co-authored-by: achatterjee-grafana <aparajita.chatterjee@grafana.com>

davidmparrott committed:
Alerting: Return cached alerts for prometheus/api/v1/alerts (#32654)

* Return cached alerts for prometheus/api/v1/alerts

* Return not implemented for /prometheus/grafana/api/v1/rules

* Set StartsAt for already alerting states

* Fix tests

leeoniya committed:
Data: frame compare refactor, drop unused skipProperties (#32700)

Clarity-89 committed:
Search: update item padding (#32701)

hugohaggmark committed:
Variables: Confirms selection before opening new picker (#32586)

* Variables: Confirms selection before opening new picker

* Tests: fixes e2e tests

torkelo committed:
Babel: @babel/plugin-transform-react-constant-elements to babel config to optimize react performance (#32629)

shuotli committed:
AzureMonitor: Add deep links to Azure Portal for Metrics (#32273)

* Add Explore in azure portal functionality to Grafana AM data sources for Metrics

* fix url

* fix build

* Fix the URL with metric and aggregation

* user new version of metric blade

* use lookup object instead of enum for converting aggregation type

Co-authored-by: joshhunt <josh@trtr.co>

gabor committed:
Datasource config: correctly remove single custom http header (#32445)

* grafana-ui: data-source-settings: fix remove-last-http-header case

* adjust code to not-mutate props-data

* improved tests and testability

* datasource: custom-http-headers: cleanup secure-values too

Clarity-89 committed:
Card: add missing padding (#32702)

joshhunt committed:
AzureMonitor: fix migration error from older versions of App Insights queries (#32372)

* AzureMonitor: fix nill reference error with missing timeGrain

* add a test

Co-authored-by: Marcus Efraimsson <marcus.efraimsson@gmail.com>

oscarkilhed committed:
Normalize color string before using them as IDs (#32713)

Commit:
Elasticsearch: Fix bucket script variable duplication in UI (#32705)

* Elasticsearch: Fix bucket script variable duplication

* Generate names based on previous value & fix list key

* Fix typos

Co-authored-by: Piotr Jamróz <pm.jamroz@gmail.com>

Co-authored-by: Piotr Jamróz <pm.jamroz@gmail.com>

jackw committed:
chore: Bump storybook to 6.2 (#32638)

* chore(grafanaui): bump storybook to 6.2, remove nohoist babel conflict hack

* chore(grafanaui): update storybook react doc gen defaults

* feat(grafanaui): map public/img as storybook static path for images/icons

* refactor(grafanaui): remove storybook noopControl in favour of parameters.controls.exclude

* feat(grafanaui): add component for storybook docs theming

* chore(grafanaui): silence button docs react warnings

* chore(grafanaui): bump storybook to v6.2.3

* revert(grafanaui): remove test description param from legacy switch story

torkelo committed:
Plugins: Support es5 plugins extending es6 core classes (#32664)

* Plugins: Support es5 plugins extending es6 core classes

* Make all base classes backward compatible

torkelo committed:
Emotion: Remove emotion import (#32715)

alexanderzobnin committed:
Access Control: move features to Enterprise (#32640)

* Move db package WIP

* Implement OSS access control

* Register OSS access control

* Fix linter error in tests

* Fix linter error in evaluator

* Simplify OSS tests

* Optimize builtin roles

* Chore: add comments to the exported functions

* Remove init from ossaccesscontrol package (moved to ext)

* Add access control as a dependency for http server

* Modify middleware to receive fallback function

* Middleware: refactor fallback function call

* Move unused models to enterprise

* Simplify AccessControl type

* Chore: use bool IsDisabled() method instead of CanBeDisabled interface

kminehart committed:
add redis integration test for enterprise (#32617)

papagian committed:
[Alerting]: Require login for alerting API routes (#32688)

* [Alerting]: Require login for alerting API routes

* Fix example requests

* [Alerting]: Add /api prefix to all routes (#32716)

zoltanbedi committed:
Prometheus: Fix instant query to run two times when exemplars enabled (#32508)

* Prometheus: Fix instant query to run two times when exemplars enabled

* Update exemplar messages

* Tempo: show empty response if response is empty

papagian committed:
Docs: Rephrase Alert execution section in Alerting (#32631)

* Docs: Rephrase Alert execution section in Alerting

* Small fix

datasources -> data sources

* Rephrase alert execution

09jvilla committed:
Merge branch 'index_update' of github.com:grafana/grafana into index_update

09jvilla committed:
Making updates to copy, plus rebasing to resolve merge conflict.

09jvilla commented:
Closing this and reopening a clean PR -- seems like I kicked off a much more wide ranging review when I had to rebase.
